src/test/java/com/exasol/adapter/dialects/exasol/AbstractExasolSqlDialectIT.java

```diff
@@ -853,14 +840,13 @@ void testCurrentClusterFunction() throws SQLException {
         }
     }
 
-    void requireMajorVersion(final int required) {
-        assumeTrue(isMajorVersionOrHigher(required), "Required fix for this test is only available with Exasol version "
-                + required + " or later, " + EXASOL.getDockerImageReference() + " is not supported.");
-    }
-
-    boolean isMajorVersionOrHigher(final int majorVersion) {
+    boolean isVersionOrHigher(final int majorVersion, final int minorVersion, final int fixVersion) {
         final ExasolDockerImageReference version = EXASOL.getDockerImageReference();
-        return version.hasMajor() && (version.getMajor() >= majorVersion);
+        final long comparableImageVersion = (version.hasMajor() ? version.getMajor() * 1000000L : 0)
+                + (version.hasMinor() ? version.getMinor() * 1000L : 0)
+                + (version.hasFix() ? version.getFixVersion() : 0);
+        final long comparableRequiredVersion = majorVersion * 1000000L + minorVersion * 1000L + fixVersion;
```

Comment:
Extract the code to calculate the comparable version into a method to avoid code duplication

Reply:
Done.